stock_manager/utils/enums.py

```diff
@@ -24,19 +37,66 @@ class _PageDetails:
 
 
 class ExportTypes(Enum):
+    """
+    Represents supported export file types.
+    
+    Members:
+    
+    - PDF: Portable Document Format
+    - CSV: Comma-Separated Values
+    - TSV: Tab-Separated Values
+    - PSV: Pipe-Separated Values
+    """
+    
     PDF = 'pdf'
     CSV = 'csv'
     TSV = 'tsv'
     PSV = 'psv'
 
 
 class StockStatus(Enum):
+    """
+    Represents inventory status for an item.
+
+    Members:
+    
+    - IN_STOCK: The item is available in sufficient quantity.
+    - LOW_STOCK: The item is running low and may need to be reordered/restocked.
+    - OUT_OF_STOCK: The item is currently unavailable.
+    """
+    
     IN_STOCK = 'In Stock'
     LOW_STOCK = 'Low Stock'
     OUT_OF_STOCK = 'Out Of Stock'
 
 
 class DatabaseUpdateType(Enum):
+    """
+    Represents the type of database modification being made.
+
+    Members:
+    
+    - ADD: Adding a new entry.
+    - EDIT: Modifying an existing entry.
+    - REMOVE: Deleting an entry.
+    """
+    
     ADD = auto()
     EDIT = auto()
     REMOVE = auto()
+
+
+class Hutches(Enum):
+    """Represents hutches at SLAC."""
+    
+    CXI = 'CXI (Coherent X-ray Imaging)',
+    MEC = 'MEC (Matter in Extreme Conditions)',
+    MFX = 'MFX (Macromolecular Femto-Second Crystallography)',
+    QRIXS = 'qRIXS',
```

Comment:
You should know that there's also chemrixs (cRIXS?), though often we lump them together as RIX because they are in the same room (and apparently three letter acronyms are important.)